rule.proto

```diff
@@ -0,0 +1,221 @@
+syntax = "proto2";
+package dv360_excel_plugin;
+
+/* A message that represents a scheduled rule for manipulating DV360 entities.
+ */
+message Rule {
+  /* required
+   * The name of the rule, set by the user.
+   */
+  optional string name = 1;
+
+  /* required
+   * The id that uniquely identified each rule.
+   */
+  optional int32 id = 2;
+
+  // required
+  optional Action action = 3;
+
+  // required
+  optional Condition condition = 4;
```

Comment:
I changed the design so that each rule only has one optional condition.

Reply:
You can remove the optional comment if the field is optional. The idea is use 
```
// required
optional field = 1
```
to replace 
```
required field = 1
```
Here is the [comment](https://github.com/DV360-spreadsheet-plugin/dv360-excel-plugin/pull/30#discussion_r444429947) i received from Ruben regarding this.